generation/new_client/new-client.py

```diff
@@ -339,32 +287,12 @@ def generate(
         cwd=monorepo_root,
     )
 
-    # Add the files to commit
-    subprocess.check_call([
-        "git", "add", "pom.xml", "gapic-libraries-bom/pom.xml",
-        "release-please-config.json", ".release-please-manifest.json"],
-        cwd=monorepo_root)
-
-    subprocess.check_call(
-        ["git", "commit", "-m",
-         f"build: add the {api_shortname} module to monorepo"],
-        cwd=monorepo_root
-    )
-
-    # Remove irrelevant changes in other modules
-    subprocess.check_call(
-        ["git", "checkout", "--", "."],
-        cwd=monorepo_root
-    )
-
-    # It seems generate_release_please_config.sh is not ready to run as
-    # part of client library generation process.
-
     print(f"Prepared new library in {workdir}")
-    print(f"Please create a pull request from that directory:\n"
-          f"  $ cd {monorepo_root}\n"
+    print(f"Please create a pull request:\n"
+          f"  $ git checkout -b new_module_java-{output_name}\n"
+          f"  $ git add .\n"
```

Comment:
This would include any change to `new-client.py`. Can you add an instruction to avoid that? With that instruction, next person who touches new-client.py will avoid committing changes to the new library creation request.

Reply:
Why would someone need to modify `new-client.py` as part of a routine new library generation? However, even if they do, what's wrong with including it with the new library PR? It could help see the impact of the changes. We've been doing this kind of thing with every other script in `generation` directory, often including changes to scripts and their execution in the same PR.